tests/unit/docs/test_client.py

```diff
@@ -51,26 +51,26 @@ def test_document_client(self):
             ]
         )
         self.assert_contains_lines_in_order(
-            ['.. py:method:: can_paginate(operation_name)'],
+            ['.. py:method:: MyService.Client.can_paginate(operation_name)'],
```

Comment:
This looks a little strange to me without seeing the context around it. Which page would we be generating methods like this on?